M2/Macaulay2/tests/normal/testing.m2

```diff
@@ -7,14 +7,17 @@ loadPackage("TestPackage", FileName => testpkg)
 check "TestPackage"
 pkgtest = tests(0, "TestPackage")
 assert instance(pkgtest, TestInput)
-assert Equation(toSequence locate pkgtest, (testpkg, 3, 1, 4, 1,,))
-assert Equation(toString pkgtest, testpkg | ":3:1-4:1")
-assert Equation(net pkgtest, "TestInput[" | testpkg | ":3:1-4:1]")
-expectedCode =  " -- " | toAbsolutePath testpkg |
-	":4:1: location of test code" | newline | "assert Equation(1 + 1, 2)"
-assert Equation(code pkgtest, expectedCode)
-assert Equation(code 0, expectedCode)
-
+assert Equation(toSequence locate pkgtest, (testpkg, 3, 5, 3, 32, 3, 5))
```

Comment:
This happened in a brew build:
```m2
 -- running   check(357, "Core")              -- .380111s elapsed
/opt/homebrew/Cellar/macaulay2/1.24.11-rc1/share/doc/Macaulay2/Core/tests/testing.m2:1:1 error:
 -- i5 : pkgtest = tests(0, "TestPackage")
 -- 
 -- o5 = TestInput[../../M2-93313-0/0.m2:3:5-3:32]
 -- 
 -- o5 : TestInput
 -- 
 -- i6 : assert instance(pkgtest, TestInput)
 -- 
 -- i7 : assert Equation(toSequence locate pkgtest, (testpkg, 3, 5, 3, 32, 3, 5))
 -- stdio:10:6:(3): error: assertion failed:
 -- (../../M2-93313-0/0.m2, 3, 5, 3, 32, 3, 5) == (/private/tmp/M2-93313-0/0.m2, 3, 5, 3, 32, 3, 5) is false
 -- 
stdio:1:5:(3): error: test(s) #333 of package Core failed.
```

Reply:
Oh, that makes sense.  Calling `minimizeFilename` should fix it.  I'll open a PR.